.github/workflows/dns-apply.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - uses: cachix/install-nix-action@d1ca217b388ee87b2507a9a93bf01368bde7cec2 # v31
       - name: dnscontrol push
         env:
+          GANDI_TOKEN: "${{ secrets.GANDI_TOKEN }}"
```

Comment:
Would be useful to add the gandi expiration date here in case we forget to update it. Jonas said a year?

Reply:
Comment added.